diff -r dea00b569fcb -r bb068e7a26e4 pub/scripts/dialog.js
--- a/pub/scripts/dialog.js	Tue Mar 24 13:22:17 2009 +0100
+++ b/pub/scripts/dialog.js	Tue Mar 31 12:05:37 2009 +0200
@@ -95,3 +95,6 @@
     Element.hide($$('.graybox')[0]);
 }
 
+function repositionDialog () {
+    Position.EyeLevel($$('.dialog')[0]);
+}
diff -r dea00b569fcb -r bb068e7a26e4 pub/scripts/weblocks.js
--- a/pub/scripts/weblocks.js	Tue Mar 24 13:22:17 2009 +0100
+++ b/pub/scripts/weblocks.js	Tue Mar 31 12:05:37 2009 +0200
@@ -74,6 +74,8 @@
     }
 
     execJsonCalls(json['on-load']);
+
+    repositionDialog();
 }
 
 function execJsonCalls (calls) {
